code/game/objects/structures/watercloset.dm

```diff
@@ -15,7 +15,18 @@
 	. = ..()
 	open = round(rand(0, 1))
 	update_icon()
-
+	
+/obj/structure/toilet/contraband/Initialize(mapload)
```

Comment:
It might be a bit better to have it handled by a variable rather than it's own subtype, considering almost all toilets were replaced with contraband toilets in the map changes. I can't think of many situations where a contraband toilet in place of the normal one would be bad.